contribute-to-tidb/review-a-pr-stage.md

```diff
@@ -0,0 +1,233 @@
+
+If a PR deletes or deprecates code, it should consider whether the documentation should also be deleted.
+
+If documentation is missing, the reviewer should ask for it.
+
+#### Error Handling
+
+Be careful with error or exception handling. This is one of the most common places for bugs.
+
+* Does the test cover the error code path?
+* Does a panic make the whole process exit for server services?
+* Does the code handle the resource releasing properly?
+* Especially, for code including concurrent goroutines, is there a goroutine leak or memory leak after the error?
+
+Sometimes it is difficult for a real program to run into the error code path, and even more difficult to reproduce and debug if it really happens. Then it is necessary to mock the error code path. Tools like [failpoint](https://github.com/pingcap/failpoint) are available for that purpose.
+
+### Performance
+
+Does the PR impose any performance impact? Sometimes a minor change on the hot path that seems unrelated may still affect performance. Is the algorithm in the PR suitable for that case?
+
+## How to write code review comments
+
+### Be respectful to the reviewees
+
+Good reviewers are compassionate. Your code always need to be reviewed. And also you always need to review other contributors' code. Regard reviews as a learning process, and everyone wins.
+
+### Ask questions rather than make statements
+
+Wording is very important. Try to be constructive in your feedback, rather than critical. You can do this by asking questions, rather than making statements. And remember to give praise alongside your constructive feedback.
+
+### Offer sincere praise
+
+Most reviewers focus only on what’s wrong with the code, but they should offer encouragement and appreciation for good practices, as well. It’s sometimes even more valuable, in terms of mentoring, to tell a developer what they did right than to tell them what they did wrong.
+
+For example, imagine you’re reviewing for an author who struggles to write documentation, and you come across a clear, concise function comment. Let them know they nailed it. They will improve faster if you tell them when they got it right instead of just waiting to ding them when they screw up.
+
+#### Summary
+
+* Be kind to the coder, not to the code.
+* Ask questions rather than make statements.
+* Treat people who know less than you with respect, deference, and patience.
+* Remember to praise when the code quality exceeds your expectation.
+* It isn't necessarily wrong if the coder's solution is different from yours.
+* Refer to the code style document when necessary.
+
+## Review with the @ti-chi-bot
+
+The TiDB community is using a service called [@ti-chi-bot](https://book.prow.tidb.io/#/en/) to help with the review of the pull requests.
+
+You can see the full list of commands accepted by the bot in [this page](https://prow.tidb.io/command-help?repo=pingcap%2Ftidb).
+
+Here are the most frequent commands you are like gonna use during a review.
```

Comment:
Did you mean "you're gonna like" or "you're gonna use"?

... during a review:

Reply:
```suggestion
Here are the most frequent commands you're gonna use during a review:
```